presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -37,20 +43,30 @@
 import static com.teradata.tempto.fulfillment.table.hive.InlineDataSource.createResourceDataSource;
 import static com.teradata.tempto.fulfillment.table.hive.InlineDataSource.createStringDataSource;
 import static com.teradata.tempto.query.QueryExecutor.query;
+import static java.lang.Math.min;
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+import static org.testng.Assert.assertEquals;
 
 public class TestShowPartitions
         extends ProductTest
         implements RequirementsProvider
 {
+    // We use hive.max-partitions-per-scan=100 in tests, so this many partitions is too many
+    private static final int TOO_MANY_PARTITIONS = 105;
+
     private static final String PARTITIONED_TABLE = "partitioned_table";
+    private static final String PARTITIONED_TABLE_WITH_VARIABLE_PARTITIONS = "partitioned_table_with_variable_partitions";
 
     @Inject
     private MutableTablesState tablesState;
 
     @Override
     public Requirement getRequirements(Configuration configuration)
     {
-        return compose(mutableTable(partitionedTableDefinition(), PARTITIONED_TABLE, MutableTableRequirement.State.CREATED));
+        return compose(
+                mutableTable(partitionedTableDefinition(), PARTITIONED_TABLE, MutableTableRequirement.State.CREATED),
```

Comment:
static import

Reply:
For `CREATED`? It doesn't look unambiguous.

Reply:
No. It is used in `mutableTable(...)` so it is known what it is. The problem could be if you would use `mutableTable(..)` together with `immutableTable(...)`.

Reply:
Okay. I prefer to leave this as-is.